release_test.go

```diff
@@ -1617,3 +1617,340 @@ func Test_Release_Version(t *testing.T) {
 		}
 	}
 }
+
+func Test_Releases_GetNewestRelease(t *testing.T) {
+	testCases := []struct {
+		Releases        []Release
+		ExpectedRelease Release
+		ErrorMatcher    func(err error) bool
+	}{
+		// Test 0 ensures that a nil list throws an execution failed error.
+		{
+			Releases:        nil,
+			ExpectedRelease: Release{},
+			ErrorMatcher:    IsExecutionFailed,
+		},
+
+		// Test 1 ensures that the newest bundle can be found.
+		{
+			Releases: []Release{
+				{
+					bundles:    []Bundle{},
+					changelogs: []Changelog{},
+					components: []Component{
+						{
+							Name:    "calico",
+							Version: "1.1.0",
+						},
+						{
+							Name:    "kubernetes",
+							Version: "1.7.5",
+						},
+					},
+					deprecated: false,
+					timestamp:  "1970-01-01T00:00:10.000000Z",
+					version:    "0.1.0",
+				},
+			},
+			ExpectedRelease: Release{
+				bundles:    []Bundle{},
+				changelogs: []Changelog{},
+				components: []Component{
+					{
+						Name:    "calico",
+						Version: "1.1.0",
+					},
+					{
+						Name:    "kubernetes",
+						Version: "1.7.5",
+					},
+				},
+				deprecated: false,
+				timestamp:  "1970-01-01T00:00:10.000000Z",
+				version:    "0.1.0",
+			},
+			ErrorMatcher: nil,
+		},
+
+		// Test 2 is the same as 1 but with different bundles.
```

Comment:
I guess the comments have copy and paste issues :D

Reply:
Done-